octopusdeploy/resource_account.go

```diff
@@ -66,6 +67,15 @@ func resourceAccount() *schema.Resource {
 				Type:     schema.TypeString,
 				Optional: true,
 			},
+			// AWS
```

Comment:
Are the changes to this file still required? Should we remove `resource_account.go`?

Reply:
I removed those as they are no longer needed @droyad